tests/config.json

```diff
@@ -1,12 +1,12 @@
 {
   "ethernet": [
   {
-    "dictionary": "//awe-srv-max-prd/distext/products/CAP-XCR/firmware/2.4.0/cap-xcr-c_eth_2.4.0.xdf",
-    "ip": "192.168.2.11",
+    "dictionary": "//awe-srv-max-prd/distext/products/EVE-XCR/firmware/2.4.0/eve-xcr-c_eth_2.4.0.xdf",
+    "ip": "192.168.2.10",
     "port": 1061,
     "save_config_file": "ethernet-config.xcf",
-    "load_config_file": "//azr-srv-ingfs1/dist/setups/setup_cap_can/1.1.0/config.xml",
-    "fw_file": "//awe-srv-max-prd/distext/products/CAP-XCR/firmware/2.4.0/cap-xcr-c_2.4.0.lfu"
+    "load_config_file": "//azr-srv-ingfs1/dist/setups/setup_eve_can/1.2.0/config.xml",
+    "fw_file": "//awe-srv-max-prd/distext/products/EVE-XCR/firmware/2.0.0/eve-xcr-c_2.4.0.sfu"
```

Comment:
FW file path is wrong version folder should be 2.4.0

Reply:
Fixed [0795c44](https://github.com/ingeniamc/ingenialink-python/pull/424/commits/0795c44d91d3fe03d37c3995e047f41baad68f42)[0795c44](https://github.com/ingeniamc/ingenialink-python/pull/424/commits/0795c44d91d3fe03d37c3995e047f41baad68f42)